.excludelint

```diff
@@ -1,4 +1,5 @@
 (vendor/)
 (generated/)
 (_mock.go)
+(_mock_test.go)
```

Comment:
This btw, is to allow us to generate private mocks purely within testing scope; I don't want to export some mocks.